Documentation/upgrade.md

```diff
@@ -20,6 +20,10 @@ With this manual upgrade guide, there are a few notes to consider:
 * **WARNING:** Upgrading a Rook cluster is a manual process in its very early stages.  There may be unexpected issues or obstacles that damage the integrity and health of your storage cluster, including data loss.  Only proceed with this guide if you are comfortable with that.
 * Rook is still in an alpha state.  Migrations and general support for breaking changes across versions are not supported or covered in this guide.
 * This guide assumes that your Rook operator and its agents are running in the `rook-system` namespace. It also assumes that your Rook cluster is in the `rook` namespace.  If any of these components is in a different namespace, search/replace all instances of `-n rook-system` and `-n rook` in this guide with `-n <your namespace>`.
+* Rook cluster installations on Kubernetes prior to version 1.7.x, use [ThirdPartyResource](https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-third-party-resource/) that have been deprecated as of 1.7 and removed in 1.8. When upgrading your Kubernetes cluster Rook TPRs have to be migrated to CustomResourceDefinition (CRD) following [Kubernetes documentation](https://kubernetes.io/docs/tasks/access-kubernetes-api/migrate-third-party-resource/). Rook TPRs that require migration during upgrade are: 
```

Comment:
I feel this needs to go to a different section. This is about rook upgrade. The case where TPR migration is relevant is during a K8S upgrade.

Reply:
@kokhang I agree, but didn't see a more reasonable place, do you have a suggestion for a better location?

I didn't want to create a new documentation section or a page. The  *Considerations* section in Upgrade doc sounded like the best option.

Reply:
How about creating a new sub heading under this same doc. Title it "Upgrade Kubernetes" and make it same level as "Upgrade Rook Software"